src/main/flight/pid.c

```diff
@@ -281,7 +281,7 @@ void pidUpdateTpaFactor(float throttle)
     static bool isTpaLowFaded = false;
     // don't permit throttle > 1 & throttle < 0 ? is this needed ? can throttle be > 1 or < 0 at this point
     throttle = constrainf(throttle, 0.0f, 1.0f);
-    bool isThrottlePastTpaLowBreakpoint = (throttle < pidRuntime.tpaLowBreakpoint && pidRuntime.tpaLowBreakpoint > 0.01f) ? false : true;
+    bool isThrottlePastTpaLowBreakpoint = throttle >= pidRuntime.tpaLowBreakpoint || pidRuntime.tpaLowBreakpoint <= 0.01f;
```

Comment:
Might be better to leave this refactoring for 4.6, @limonspb ?

Reply:
done